templates/CRM/Osdi/Page/OSDIJob.tpl

```diff
@@ -207,7 +202,7 @@
             }
             if (!edit) {
                 // redirect to mapping
-                window.location.href = "/civicrm/osdi/mapping?change=1&endpoint=" + rootendpoint.val();
+                window.location.href = "/civicrm/osdi/mapping?change=1&endpoint=" + peopleendpoint.val();
```

Comment:
note: REDO THIS url redirect - anyone know if there's some utility to grab the URL's by platform?